src/material/table/BUILD.bazel

```diff
@@ -31,14 +31,26 @@ ng_module(
 
 sass_library(
     name = "table_scss_lib",
-    srcs = glob(["**/_*.scss"]),
+    srcs = glob(
+        ["**/_*.scss"],
+        exclude = ["_table-flex-styles.scss"],
+    ),
```

Comment:
Rather than using a glob with an exclude, could use specify the exact files (I think just `_table-theme.scss`)? I think we'd typically only use this pattern in situations where there are more files to capture